.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,60 @@
+
+name: 🐞 Bug
+description: Report an issue to help improve the project.
+title: "[BUG] <title>"
+labels: ["bug"]
```

Comment:
If you can see labels panel in GitHub UI, it is editable. It can be edited while creating or after.
since this is bug report template purpose of this `labels:["bug"]` is to tag/label issue as a "bug", later when we will have features template we will use `labels: ["enhancement"]`. This is kind of auto triaging issues created will land with proper labels by default.

Reply:
When I look at our [existing page](https://github.com/distributed-system-analysis/pbench/issues/new), I see gear icons next to each of "Assignees", "Labels", "Projects", and "Milestones"; when I look at your [example page](https://github.com/vishalvvr/pbench/issues/new/choose), none of them have gear icons displayed.

A single issue typically carries several labels, even at creation/triage (e.g., looking at the first page of our issues list, other than the untriaged issues, there's only one issue with fewer than three labels).  Typically, they are marked "bug"/"enhancement", "Agent"/"Server", and then they have one or more additional labels indicating their component or area.

I'm not saying that this is "the right way" to do it, but it is different enough from what the form is providing that it seems like having the form applying labels may be counter-productive:  for instance, it will be harder to spot untriaged issues, because all issues (from the proposed form) will already have the "bug" label already applied.

If we really need the form to apply a label, perhaps it would be better to have it apply a "needs triage" label instead.

Reply:
> When I look at our [existing page](https://github.com/distributed-system-analysis/pbench/issues/new), I see gear icons next to each of "Assignees", "Labels", "Projects", and "Milestones"; when I look at your [example page](https://github.com/vishalvvr/pbench/issues/new/choose), none of them have gear icons displayed.

That's because you are not added to my project(fork) as contributor/owner and for pbench(official) repo you are, but that's an expected behavior.

if you want to try this without template try [link](https://github.com/vishalvvr/pbench/issues/new)
you still wont be able to see label edit option :)

> I'm not saying that this is "the right way" to do it, but it is different enough from what the form is providing that it seems like having the form applying labels may be counter-productive: for instance, it will be harder to spot untriaged issues, because all issues (from the proposed form) will already have the "bug" label already applied.

True 

> If we really need the form to apply a label, perhaps it would be better to have it apply a "needs triage" label instead.

Sound good option to me.

Reply:
I like the idea of a default `needs-triage` label to mark something as "incoming".

Reply:
> That's because you are not added to my project(fork) as contributor/owner and for pbench(official) repo you are, but that's an expected behavior.

Agreed.  So, if, when a contributor/owner is using this form, s/he'll be able to use the right-hand sidebar, then that allays most of my concerns, but I think having the form mark the issue as `needs-triage` rather than `bug` would be better.